docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -20,9 +20,9 @@ To complete this guide, you need:
 
 * less than 15 minutes
 * an IDE
-* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* `JDK 8 or 11+` installed with `JAVA_HOME` configured appropriately
 * A xref:configuring-c-development[working C development environment]
-* GraalVM version {graalvm-version} installed and  xref:configuring-graalvm[configured appropriately]
+* `GraalVM` version `{graalvm-version}` installed and xref:configuring-graalvm[configured appropriately]
```

Comment:
Why backticks around GraalVM? It's a name/noun, not a code snippet?

Reply:
Mentioned in PR description

> Highlighted GraalVM and JDK version so it's more visible on https://quarkus.io/guides/building-native-image-guide page

Reply:
Version, yes! Name, why? I know, retentive.. but if you want emphasis, making the name *bold* is more appropriate than backticks. :-P

Reply:
Ok, will use bold. I originally picked backticks because`<code>...</code>` in the final html stands out a bit more.